server/schema/auth/updateUser.js

```diff
@@ -19,6 +19,15 @@ const schema = {
       maxLength: 10,
       pattern: '^[0-9]{10}$',
     },
+    department: {
+      type: 'number',
+    },
+    branch: {
+      type: 'number',
+    },
+    bio: {
+      type: 'string',
+    },
```

Comment:
bio should allow maximum 100 characters only
So, apply  `maxlength: 100`
`

Reply:
done